### PR TITLE
Do/guide revamp

### PR DESCRIPTION
**Pull request by Foff**
Preparing the guide for release

### CODE REVIEW

File: user-guide/source/guide/getting-help.rst [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+*Forum*: when you have a question about the platform, try posting
+it on our forum. You can find it on forum.genestack.org. Our community
+will definitely be keen to help you, and if not, our team regularly checks the forum and will answer any unanswered questions.
+
+*Email*: you can email us at support@genestack.com 
+
+FAQ
+===
+
+**Where do I find data that was shared with me?**
```
Comment (klas-genestack) [inline user-guide/source/guide/getting-help.rst]: This should be a header, not an emphasized sentence!

File: user-guide/source/guide/getting-help.rst [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+If the files were linked into the group folder, you will find them there.
+You can access the group folder from the file browser, under the "Shared with me" section. 
+Otherwise, the files can be found via search, if you know their name or accession.
+
+**How do I reuse a data flow?**
```
Comment (klas-genestack) [inline user-guide/source/guide/getting-help.rst]: As above, so below.

File: user-guide/source/guide/getting-help.rst [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+the pipeline (in the form of a data flow) that was used to create this
+file. Read more about data flows in this tutorial_.
+
+**What is the difference between BWA and Bowtie2?**
+
+The biggest differences between the two aligners are:
+
+- the way of accepting or rejecting an alignment;
+
+BWA counts the number of mismatches between the read and the
```
Comment (klas-genestack) [inline user-guide/source/guide/getting-help.rst]: Such unindented paragraphs tear the list in two.

File: user-guide/source/guide/getting-help.rst [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+-  paired-with-unpaired (3 or 4 files locally, 2 files in Genestack).
+
+During import, Genestack recognises them and imports them in
+their respective format-free form. If the platform
+cannot recognise the files automatically, you can allocate the files
+manually.
+
+**What is the difference between an experiment and a folder?**
+
+Experiments are a special kind of folder, which can only contain
```
Comment (klas-genestack) [inline user-guide/source/guide/getting-help.rst]: Aren't experiments deprecated and superseded by Datasets™?

File: user-guide/source/guide/references.rst [hunk 5]
```diff
@@ -0,0 +1,68 @@
+References
+==========
+
+*Introduction to bioinformatics:*
```
Comment (klas-genestack) [inline user-guide/source/guide/references.rst]: Seems a header candidate to me.

File: user-guide/source/guide/references.rst [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+- Luscombe NM et al. "What is bioinformatics? An introduction and overview." Yearbook of Medical Informatics. 2001. (https://www.ebi.ac.uk/luscombe/docs/imia\_review.pdf)
```
Comment (klas-genestack) [inline user-guide/source/guide/references.rst]: Why we went for plaintext links in references section?  By the way, don't forget about inline link syntax:
```rst
Luscombe NM et al.
`"What is bioinformatics? An introduction and overview." <https://www.ebi.ac.uk/luscombe/docs/imia\_review.pdf>`_
Yearbook of Medical Informatics. 2001.
```

File: user-guide/source/guide/references.rst [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+*Application management:*
+
+- https://msdn.microsoft.com/en-us/magazine/dn451436.aspx
```
Comment (klas-genestack) [inline user-guide/source/guide/references.rst]: *Nameless* links? =\

File: user-guide/source/guide/getting-help.rst [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+When you want to use the data flow to run your analysis, on the Data
+Flow Editor page you can click on "Run Data Flow" button, which will
+take you to Data Flow Runner. Here you can not only edit source files
+and parameters, but also start initialization of your files.
+
+**How can I initialize multiple files at once?**
+
+You can use the File Initializer application for this.
+Select the files you want to initialize in File Browser, right-click on them
+and go to Manage > File Initializer. From there, click the button "Initialize all"*[]:
```
Comment (klas-genestack) [inline user-guide/source/guide/getting-help.rst]: `user-guide/source/guide/getting-help.rst:62: WARNING: Inline emphasis start-string without end-string.`

**Comment by klas-genestack** [comment]
Also I very much dislike the idea of splitting repo in **two unlinked Sphinx projects**.  How it's supposed to be built and deployed?  Why would we break **all the existing links**?

SAD!

**Comment by klas-genestack** [comment]
@daniel-ohayon 
> Also I very much dislike the idea of splitting repo in two unlinked Sphinx projects. How it's supposed to be built and deployed? Why would we break all the existing links?

To clarify: I don't know what you were trying to accomplish, so I cannot give any advice on how to fix this.  All that I see is that this branch completely breaks all the ReadTheDocs machinery already in place, without any attempt to actually organize the result.

**Comment by Maria-Alekseeva** [comment]
@klas-genestack 

> To clarify: I don't know what you were trying to accomplish, so I cannot give any advice on how to fix this. 

Ideally, we would like to see both the tutorials and the guide on the same ReadTheDocs web site, hosting all user documentation. 
Would it be possible to create just one Sphinx project including the two current ones and do not break the links to the tutorials?

Then, as for the interface of the documentation page, we would like to add our tutorials to the index on the left. So that they will have the same level as the main sections corresponding to the RST files in the folder 'guide'.

As a result, we could have the following index:

- Introduction to Bioinformatics (intro-to-bioinformatics.rst)
- Genestack Key Features (genestack-features.rst)
- Using Genestack (using-genestack.rst)
- Introduction to NGS Data Analysis (intro-to-ngs.rst)
- Pipelines and Applications (pipelines.rst)
- References (references.rst)
_<maybe some kind of separator would be nice here. Don't know what works best, maybe just a blank line as the simplest variant. What do you think?>_
- FAQ (*new, based on the getting-help.rst)
- **Tutorials**
- Getting in touch with Genestack (*new, based on the getting-help.rst)

In this case we will have to split the getting-help.rst into two: "Getting in touch with Genestack" and "FAQ". Also, the section about tutorials should probably be removed. If we add the real tutorials directly in the index, then there is no need to mention them in a separate tiny section. (If it is OK, we could probably do this by ourselves.) 

The problem with such approach is that, in this case, all the tutorials must be in just one RST file. There're lots of data. Is there any way to overcome it?

Thx

**Comment by klas-genestack** [comment]
See my proposed fix in #39.
I didn't give much thought about actual looks and organization of ToCs, but this solution will both keep links to tutorials the same, and more-or-less nicely includes tutorials into new user guide.

File: source/conf.py [hunk 9]
```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = u'Genestack User Tutorials'
-copyright = u'2017, Genestack'
+copyright = u'2016, Genestack'
```
Comment (klas-genestack) [inline source/conf.py]: Fix this back

File: source/guide/pipelines.rst [hunk 10]
```diff
@@ -0,0 +1,3053 @@
...
+3. What you can do if the quality is unsatisfactory.
+
+The metrics table gives you quick indicators as to the status of each of
+the quality metrics calculated.
+
+- Yellow triangles - warnings;
+- Red X-es - failures;
+- Green check marks in circles - ok.
+
+1. **Basic statistics**
```
Comment (klas-genestack) [inline source/guide/pipelines.rst]: These should be headers.

File: source/guide/pipelines.rst [hunk 11]
```diff
@@ -0,0 +1,3053 @@
...
+
+2. **Sequence length distribution**
+
+.. image:: images/fastqc_sequence_length_distribution.png
+
+Reports lengths of all sequences.
+
+*Warning*
+
+This report will get warnings if the lengths are not identical, but this can
```
Comment (klas-genestack) [inline source/guide/pipelines.rst]: `*Warning*: this report should be...`

File: source/guide/pipelines.rst [hunk 12]
```diff
@@ -0,0 +1,3053 @@
...
+usually be ignored, as it is expected for some sequencing platforms.
+
+3. **Per sequence GC content**
+
+.. image:: images/fastqc_per_sequence_gc_content.png
+
+For data of good quality, the graph will show a normal, bell-shaped
+distribution.
+
+*Warning*
```
Comment (klas-genestack) [inline source/guide/pipelines.rst]: See above

File: source/guide/pipelines.rst [hunk 13]
```diff
@@ -0,0 +1,3053 @@
...
+
+A warning is raised when the sum of the deviations from the normal distribution
+represents more than 15% of the reads.
+
+Warnings are usually caused by a presence of contaminants. Sharp peaks may
+represent a presence of a very specific contaminant (e.g. an adaptor). Broader
+peaks may indicate contamination with a range of contaminants.
+
+*Improving data quality*
```
Comment (klas-genestack) [inline source/guide/pipelines.rst]: Looks like a header to me.

File: source/guide/pipelines.rst (hunk 10 above)
Comment (klas-genestack) [inline source/guide/pipelines.rst]: Probably [autonumbered ones](http://docutils.sourceforge.net/docs/ref/rst/directives.html#automatic-section-numbering), too.

File: source/guide/pipelines.rst [hunk 14]
```diff
@@ -0,0 +1,3053 @@
...
+
+Ideally, we expect to see a sharp peak at the very end of the graph (meaning
+most frequently observed mean quality scores are above 27).
+
+*Warning*
+
+A warning is raised when the peak is shifted to the left, which means the most
+frequently observed mean quality is below 27. This equals to a 0.2% error rate.
+
+*Improving data quality*
```
Comment (klas-genestack) [inline source/guide/pipelines.rst]: Header?

File: source/guide/pipelines.rst [hunk 15]
```diff
@@ -0,0 +1,3053 @@
...
+contains widely used PCR primers and adaptors for Illumina, ABI etc. (see the
+`list of primers and adaptors`_ we remove).
+
+.. _list of primers and adaptors: https://s3.amazonaws.com/bio-test-data/Genestack_adapters.txt
+
+The occurrence threshold before adapter clipping is set to 0.0001. It refers to
+the minimum number of times an adapter needs to be found before clipping is
+considered necessary.
+
+1. **Minimum length of the trimmed sequence (bp)**. The application will
```
Comment (klas-genestack) [inline source/guide/pipelines.rst]: Why it's numbered?  I don't see point 2.

File: source/guide/pipelines.rst [hunk 16]
```diff
@@ -0,0 +1,3053 @@
...
+To identify highly variable genes you can try different options:
+
+1. The **Use spike-ins to calibrate noise** option determines whether or not
+   spike-in data should be taken into account. If you select only one folder
+   before running the application, you will use spike-free algorithm and this
+   option will be switched off by default. But if you select two folders, one
+   for biological and the other for spike-in data, you can use the Brennecke
+   algorithm which requires this option.
+2. The **Exclude samples with low coverage** option allows you to exclude or
+   include for analysis samples with low read counts. (default: checked
```
Comment (klas-genestack) [inline source/guide/pipelines.rst]: 3k line file?! It should, no, it *has to* be split!

File: source/guide/using-genestack.rst [hunk 17]
```diff
@@ -0,0 +1,1137 @@
...
+
+.. image:: images/export2.png
+
+The application creates an Export File that contains a special link to download
+the selected assays, that is accessible until the corresponding Export File is deleted.
+
+.. image:: images/export3.png
+
+**Sharing the link** enables your collaborator to download data even if they do not have
+a Genestack account. All the Export files are stored in the “Exports” folder.
```
Comment (klas-genestack) [inline source/guide/using-genestack.rst]: This is a bit too big for me as well.

File: source/guide/references.rst [hunk 18]
```diff
@@ -0,0 +1,85 @@
+References
+==========
+
+Introduction to bioinformatics
+------------------------------
+
+- Luscombe NM et al. "What is bioinformatics? An introduction and overview." Yearbook of Medical Informatics. 2001; https://www.ebi.ac.uk/luscombe/docs/imia\_review.pdf
```
Comment (klas-genestack) [inline source/guide/references.rst]: Say NO to nameless links!

File: source/guide/pipelines.rst (hunk 11 above)
Comment (Foff) [inline source/guide/pipelines.rst]: ok. I will do the same for "Failure" and "Improving data quality".

File: source/guide/pipelines.rst (hunk 12 above)
Comment (Foff) [inline source/guide/pipelines.rst]: ok.

File: source/guide/pipelines.rst (hunk 13 above)
Comment (Foff) [inline source/guide/pipelines.rst]: Nope, why header? It's just about what to do if you get warnings or failures in the report. We get "warning", "failure" and "improving data quality" sections for each module in FastQC report.

Maybe to make it look like for warnings (you suggested above):

*Improving data quality*: bla-bla-bla...

File: source/guide/pipelines.rst (hunk 15 above)
Comment (Foff) [inline source/guide/pipelines.rst]: Agree, will fix it.

File: source/guide/pipelines.rst (hunk 16 above)
Comment (Foff) [inline source/guide/pipelines.rst]: Do not understand. What do you mean here?

**Pull request merged**